### DISCUSSION TITLE
Adding Sku in the item

### DISCUSSION THREAD

**Discussion by wolfcoder** [woocommerce/woocommerce#50426] [Checkout Flow]
How add SKU in the item cart?

**Comment by mikejolley** [comment]
It's not possible to add the SKU to the display of cart line items in the cart block. If you'd like to have this functionality I'd recommend posting a feature request with details of your use case https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=type%3A+enhancement&projects=&template=2-enhancement.yml&title=%5BEnhancement%5D%3A+

Thanks!

**Reply by wolfcoder** [reply]
Thank you for the suggestion. I truly appreciate it. I'll look into it further.
